example.settings.json

```diff
@@ -12,9 +11,17 @@
 	"discord": {
 		"auth": {
 			"token": "BOT TOKEN HERE"
-		},
-		"channelID": "DISCORD CHANNEL ID HERE",
-		"serverID": "SERVER (GUILD) ID HERE"
+		}
 	},
+	"chatMap": [
```

Comment:
I think `bridgeMap` will be a more descriptive name. Of course the rest of the code will also have to reflect this if changed